webserver/observe/metrics/src/test/java/io/helidon/webserver/observe/metrics/TestInvalidAcceptType.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.webserver.observe.metrics;
+
+import java.util.List;
+
+import io.helidon.common.testing.http.junit5.SocketHttpClient;
+import io.helidon.http.Method;
+import io.helidon.webserver.testing.junit5.ServerTest;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@ServerTest
+class TestInvalidAcceptType {
+
+    private final SocketHttpClient client;
+
+    TestInvalidAcceptType(SocketHttpClient client) {
+        this.client = client;
+    }
+
+    @Test
+    void checkResponseCode406() {
+        String response = client.sendAndReceive(Method.GET,
+                "/observe/metrics",
```

Comment:
This issue also exists when accessing a user define handler like "/greet" -- sorry, didn't mention that in the original bug. Can you try that use case too?

Reply:
Sure, let me try that too